templates/web/base/admin/report-assignment.html

```diff
@@ -0,0 +1,15 @@
+[%  IF c.user.has_permission_to('report_inspect', c.cobrand.body.id ) %] 
+    [%# see report's assigned user %]
+    <label class="inline-text" for="assignment">[% loc('Assigned to:') %]</label>
```

Comment:
I'm not sure this should be a label given there's no input element.

Reply:
A label has to be associated with a form control as it says at the link you provide, there's no form control present here, so it should not be a label.